src/main/java/io/vertx/core/shareddata/Shareable.java

```diff
@@ -12,18 +12,26 @@
 package io.vertx.core.shareddata;
 
 /**
- * A marker interface which allows you to put arbitrary objects into a {@link io.vertx.core.shareddata.LocalMap}.
+ * An interface which allows you to put arbitrary objects into a {@link io.vertx.core.shareddata.LocalMap}.
  * <p>
  * Normally local maps only allow immutable objects or other copiable objects such as {@link io.vertx.core.buffer.Buffer}
  * instances in order to avoid shared access to mutable state.
  * <p>
- * However if you have an object that you know is thread-safe you can mark it with this interface and then you
- * will be able to add it to {@link io.vertx.core.shareddata.LocalMap} instances.
+ * However if you have a mutable object that you want to store in a {@link io.vertx.core.shareddata.LocalMap}
```

Comment:
Same here, the previous paragraph will be confusing after this change, since it's now also a `Shareable` object.